packages/lesswrong/components/votes/lwReactions/ReactionsAndLikesVote.tsx

```diff
@@ -5,6 +5,7 @@ import { useVote } from '../withVote';
 import { defineStyles, useStyles } from '@/components/hooks/useStyles';
 import type { LikesList } from '@/lib/voting/reactionsAndLikes';
 import { useCurrentUser } from '@/components/common/withUser';
+import { isAdmin } from '@/lib/vulcan-users';
```

Comment:
this should be `userIsAdmin` (I didn't even realize we had this, it looks like it's literally just the same thing reassigned to a different name?)